readers/influxdb/messages_test.go

```diff
@@ -149,10 +157,85 @@ func TestReadAll(t *testing.T) {
 			limit:  10,
 			query:  map[string]string{"subtopic": subtopic},
 			page: readers.MessagesPage{
-				Total:    uint64(len(subtopicMsgs)),
+				Total:    uint64(len(valSubtopicMsgs)),
+				Offset:   0,
+				Limit:    10,
+				Messages: valSubtopicMsgs[0:10],
+			},
+		},
+		"read message with from/to": {
+			chanID: chanID,
+			offset: 0,
+			limit:  10,
+			query: map[string]string{
+				"from": fmt.Sprintf("%f", messages[fromToNum].Time),
+				"to":   fmt.Sprintf("%f", messages[0].Time),
+			},
+			page: readers.MessagesPage{
+				Total:    fromToNum,
+				Offset:   0,
+				Limit:    10,
+				Messages: messages[1:5],
+			},
+		},
+		"read message with value": {
+			chanID: chanID,
+			offset: 0,
+			limit:  10,
+			query:  map[string]string{"value": fmt.Sprintf("%f", val)},
+			page: readers.MessagesPage{
+				Total:    msgsValNum,
+				Offset:   0,
+				Limit:    10,
+				Messages: valSubtopicMsgs[0:10],
+			},
+		},
+		"read message with v": {
+			chanID: chanID,
+			offset: 0,
+			limit:  10,
+			query:  map[string]string{"v": fmt.Sprintf("%f", val)},
+			page: readers.MessagesPage{
+				Total:    msgsValNum,
+				Offset:   0,
+				Limit:    10,
+				Messages: valSubtopicMsgs[0:10],
+			},
+		},
+		"read message with vb": {
+			chanID: chanID,
+			offset: 0,
+			limit:  10,
+			query:  map[string]string{"vb": fmt.Sprintf("%t", boolVal)},
+			page: readers.MessagesPage{
+				Total:    msgsValNum,
+				Offset:   0,
+				Limit:    10,
+				Messages: boolMsgs[0:10],
+			},
+		},
+		"read message with vs": {
+			chanID: chanID,
+			offset: 0,
+			limit:  10,
+			query:  map[string]string{"vs": stringVal},
+			page: readers.MessagesPage{
+				Total:    msgsValNum,
+				Offset:   0,
+				Limit:    10,
+				Messages: stringMsgs[0:10],
+			},
+		},
+		"read message with vd": {
+			chanID: chanID,
+			offset: 0,
+			limit:  10,
+			query:  map[string]string{"vd": dataVal},
+			page: readers.MessagesPage{
+				Total:    msgsValNum,
 				Offset:   0,
 				Limit:    10,
-				Messages: subtopicMsgs[0:10],
+				Messages: dataMsgs[0:10],
 			},
 		},
 	}
```

Comment:
You can replace all those `0` and `10` constants with constants or vars.

Reply:
This wouldn't be consistent with all other tests and I find it simpler to understand when you see the value of limit, offset for each test.

Reply:
In things tests we use `n` var for non zero limit values https://github.com/mainflux/mainflux/blob/master/things/postgres/things_test.go

Reply:
ok I can add this `n`